Lib/test/test_urlparse.py

```diff
@@ -417,6 +417,11 @@ def test_urljoins(self):
         self.checkJoin('svn+ssh://pathtorepo/dir1', 'dir2', 'svn+ssh://pathtorepo/dir2')
         self.checkJoin('ws://a/b','g','ws://a/g')
         self.checkJoin('wss://a/b','g','wss://a/g')
+        self.checkJoin(
```

Comment:
make these new SchemeFlag specific test methods instead of appending to an existing long one.